libcnb-cargo/src/main.rs

```diff
@@ -195,3 +200,26 @@ fn setup_logging() {
         std::process::exit(exit_code::UNSPECIFIED_ERROR);
     }
 }
+
+/// Recursively calculate the size of a directory and its contents in bytes.
+// Not using `fs_extra::dir::get_size` since it doesn't handle symlinks correctly:
+// https://github.com/webdesus/fs_extra/issues/59
+fn calculate_dir_size(path: impl AsRef<Path>) -> io::Result<u64> {
```

Comment:
(Optional, Observation) Since this is not tail-recursive, we might overflow the stack when the directory structure is deep. Probably nothing to worry about for a standard buildpack. Looking at `fs_extra`, their implementation isn't better in this regard, so I'm fine with keeping it as-is.